docs/CarbonData-on-docker.md

```diff
@@ -0,0 +1,72 @@
+#  CarbonData on docker
+This tutorial provides an easy way to try CarbonData. To follow along with this guide, please install [docker](https://www.docker.com/products/docker) at first.
+
+## 1 Building carbondata
+To get the jar files, we need build carbondata project. Here Spark version is 1.6.1 and Hadoop version is 2.7.2.
+```shell
+git clone https://github.com/apache/incubator-carbondata.git carbondata
```

Comment:
Why not do all these steps in the docker image? 
To let user try carbon quickly, I suggest just let user run carbonate-docker image and can start query immediately. Is there any difficulties in do that?